nerfstudio/cameras/cameras.py

```diff
@@ -778,15 +778,15 @@ def _compute_rays_for_vr180(
 
             return vr180_origins, directions_stack
 
-        for cam in cam_types:
-            if CameraType.PERSPECTIVE.value in cam_types:
```

Comment:
As an alternative fix, would it make sense to:
- Remove the `for cam in cam_types`
- De-dent all of the contents
- Change all of the `if` to `elif`?

I went through this file history and it seems like that's how these conditions were originally set up: https://github.com/nerfstudio-project/nerfstudio/blob/d413ca42042ae7b2dabdbe137c7eca7bceb42b88/nerfstudio/cameras/cameras.py#L646-L681

Which makes sense to me. The current fix also seems fine though!

Reply:
That's actually the way I originally wrote the fix, but decided to fix the loop instead since it seemed more in line with the intentions of the original author.

I don't have a strong opinion on which way is better, but I don't have a way to test anything other than fisheye and perspective cases so I figured the smaller change was safer.

Reply:
makes sense, thanks @decrispell!